modules/caddyhttp/encode/zstd/zstd.go

```diff
@@ -27,7 +27,10 @@ func init() {
 }
 
 // Zstd can create Zstandard encoders.
-type Zstd struct{}
+type Zstd struct {
+	// Compression level refer to type constants value from zstd.SpeedFastest to zstd.SpeedBestCompression
```

Comment:
Users reading the JSON docs won't know how to look at those constants. We should just list them out in the comment here for users to know what they can set.

Reply:
We should just list these strings `fastest`, `default`, `better`, `best`. Users won't look at the source code to find out which string correspond to which level.

Reply:
Were you reading the wrong commit? It has this now:

```
// List of all available levels: fastest, default, better, best
```

Reply:
My bad 😉